application/tests/org.openjdk.jmc.flightrecorder.flameview.test/META-INF/MANIFEST.MF

```diff
@@ -0,0 +1,23 @@
+Manifest-Version: 1.0
+Bundle-Version: 8.0.1.qualifier
```

Comment:
8.3.0?

Reply:
Weird that this change is here in the first place. Will take a look.